BREAKING.md

```diff
@@ -25,9 +25,40 @@ It's important to communicate breaking changes to our stakeholders. To write a g
 ### Deprecated ISummarizerRuntime batchEnd listener
 The `"batchEnd"` listener in `ISummarizerRuntime` has been deprecated and will be removed in a future release. Please remove all usage and implementations of `ISummarizerRuntime.on("batchEnd", ...)` and `ISummarizerRuntime.removeListener("batchEnd", ...)`.
 If these methods are needed, please refer to the `IContainerRuntimeBase` interface.
+
```

Comment:
These are still "2.1.0" upcoming changes? Just want to make sure you captured this under correct version (its been a little while since this PR was opened)

Reply:
Probably not. I will double check :)

Reply:
Updated to be under 2.2.0, since that should be the next release